airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/source.py

```diff
@@ -26,34 +23,6 @@
 logger = logging.getLogger("airbyte")
 
 
-class LinkedinAdsOAuth2Authenticator(Oauth2Authenticator):
-    @backoff.on_exception(
```

Comment:
Why we delete backoff policy for authenticator?

Reply:
we don't need it anymore, backoff was moved to CDK